Global variables & Variables from tests (#6023)
DataDog/documentation: content/en/synthetics/browser_tests.md

5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetics Browser test on a private URL not accessible from the public internet.
6. **How often should Datadog run the test?** Intervals are available between every 15 minutes to once per week. [Contact support][4] to enable additional frequencies for your test.

### Use variables

> You can use the [variables defined in the `Settings`][5] in the URL and Advanced Options of your browser tests. To display your list of variables, you need to type in `{{` in the field of interest.

Reviewer: ```suggestion
You can use the [variables defined in the `Settings`][5] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
```
